testing/e2e-test-utilities/src/main/kotlin/net/corda/e2etest/utilities/ClusterBuilder.kt

```diff
@@ -227,19 +276,20 @@ class ClusterBuilder(clusterInfo: ClusterInfo, val REST_API_VERSION_PATH: String
             cpbResourceName,
             getDefaultStaticNetworkGroupPolicy(groupId, staticMemberNames),
             cpiName,
-            cpiVersion
+            cpiVersion,
+            true
         )
 
     @Suppress("unused")
     /** Assumes the resource is a CPB and converts it to CPI by adding a group policy file */
     fun syncVirtualNode(virtualNodeShortId: String) =
-        post("/api/$REST_API_VERSION_PATH/maintenance/virtualnode/$virtualNodeShortId/vault-schema/force-resync", "")
+        initialClient.post("/api/$REST_API_VERSION_PATH/maintenance/virtualnode/$virtualNodeShortId/vault-schema/force-resync", "")
 
     /** Return the status for the given request id */
-    fun cpiStatus(id: String) = client.get("/api/$REST_API_VERSION_PATH/cpi/status/$id")
+    fun cpiStatus(id: String) = vNodeCreatorClient.get("/api/$REST_API_VERSION_PATH/cpi/status/$id")
 
     /** List all CPIs in the system */
-    fun cpiList() = client.get("/api/$REST_API_VERSION_PATH/cpi")
+    fun cpiList() = initialClient.get("/api/$REST_API_VERSION_PATH/cpi")
```

Comment:
I wonder if this should be done using `vNodeCreatorRole`:
```suggestion
    fun cpiList() = vNodeCreatorClient.get("/api/$REST_API_VERSION_PATH/cpi")
```